docs/get-started/Velox.md

```diff
@@ -5,28 +5,34 @@ nav_order: 1
 parent: Getting-Started
 ---
 # Supported Version
-| Type  | Version                      |
-|-------|------------------------------|
-| Spark | 3.2.2, 3.3.1                 |
-| OS    | Ubuntu20.04/22.04, Centos7/8 |
-| jdk   | openjdk8                     |
-| scala | 2.12
 
-Spark3.4.0 support is still WIP. TPCH/DS can pass, UT is not yet passed.
+| Type  | Version                         |
+|-------|---------------------------------|
+| Spark | 3.2.2, 3.3.1, 3.4.2, 3.5.1(wip) |
+| OS    | Ubuntu20.04/22.04, Centos7/8    |
+| jdk   | openjdk8/jdk17                  |
+| scala | 2.12                            |
 
-There are pending PRs for jdk11 support.
+**JDK17**
```

Comment:
Maybe, better to document this part in a common place, as it is not specific to Velox backend.

Reply:
I moved it to `NewToGluten.md`